frontend/src/scenes/dashboard/DashboardReloadAction.tsx

```diff
@@ -57,24 +57,27 @@ export function DashboardReloadAction(): JSX.Element {
                                     data-attr="auto-refresh-picker"
                                     id="auto-refresh-picker"
                                 >
-                                    <div
-                                        id="auto-refresh-check"
-                                        key="auto-refresh-check"
-                                        onClick={(e) => {
-                                            e.preventDefault()
-                                            e.stopPropagation()
-                                            setAutoRefresh(!autoRefresh.enabled, autoRefresh.interval)
-                                        }}
+                                    <Tooltip
+                                        title="Auto-refresh will only work while this tab is open"
+                                        placement="topRight"
                                     >
-                                        <Tooltip title={`Refresh dashboard automatically`} placement="bottomLeft">
+                                        <div
+                                            id="auto-refresh-check"
+                                            key="auto-refresh-check"
+                                            onClick={(e) => {
+                                                e.preventDefault()
+                                                e.stopPropagation()
+                                                setAutoRefresh(!autoRefresh.enabled, autoRefresh.interval)
+                                            }}
+                                        >
```

Comment:
The tooltip looks great! Just actually noticed we completely don't need the `div`, since the `LemonSwitch` already has `onChange`, and also the `id` here is unused. Can we just get rid of this wrapper?

Reply:
I removed the unused attributes, but the actual div needs to be there in order for the tooltip to work.

Reply:
Ohh, interesting, that _shouldn't_ be the case, but it'd be a larger change to fix that